grocery_story.R

```diff
@@ -1,6 +1,6 @@
 ## collect the variables, and pick one at random
 
-people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Miriam", "Linda", "Emmy", "Daniel")
+people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Miriam", "Linda", "Emmy", "Daniel", "Barbara")
```

Comment:
looks good, but i'm not sure how this will interact with the request to remove miriam in https://github.com/bvreede/write-a-story/pull/11